crates/shared/src/conversions.rs

```diff
@@ -20,6 +22,12 @@ pub fn u256_to_big_int(input: &U256) -> BigInt {
     BigInt::from_bytes_be(Sign::Plus, &bytes)
 }
 
+pub fn u256_to_big_uint(input: &U256) -> BigUint {
+    let mut bytes = [0; 32];
+    input.to_big_endian(&mut bytes);
+    BigUint::from_bytes_be(&bytes)
+}
```

Comment:
I copied this from `/orderbook/src/conversions` to `/shared/src/conversions`. At some point we should move everything to the shared version.